xml/art_sles_rpi_quick.xml

```diff
@@ -78,7 +78,7 @@
     </mediaobject>
    </figure>
    <figure xml:id="fig-rpi-platform-system-photo">
-    <title>Photo of the &rpi3; Model&nbsp;B Connectors, &copy; Evan-Amos / Own work / Public Domain</title>
+    <title>Photo of the &rpi3; model&nbsp;b connectors, &copy; evan-amos / own work / public domain</title>
```

Comment:
This gets messy if you follow the rule: "When words are joined by a slash, capitalize the word after the slash if the word before the slash is capitalized". It would mean "Own" becomes capitalized while "Public" does not. How about:
```suggestion
    <title>Photo of the &rpi3; model&nbsp;B connectors, &copy; Evan-Amos / Own work / Public domain</title>
```

Reply:
I think you're misapplying this rule (also, where did you get the rule from?), I would keep both "own" and "public" in lower-case

Reply:
```suggestion
    <title>Photo of the &rpi3; model&nbsp;B connectors, &copy; Evan-Amos / own work / public domain</title>
```